tests/unit-tests/test_construct_graph.py

```diff
@@ -300,6 +300,13 @@ def check_classification(res):
     res = process_labels(data, ops)
     check_classification(res)
 
+    # Check multi-label classification with invalid labels.
+    data = {'label' : np.random.randint(2, size=(13,5)).astype(np.float32)}
+    data['label'][[0, 3, 4]] = np.NAN
```

Comment:
Can you also check the case when data['label'][0][0] = np.NAN but data['label'][0][1] is a valid number? I think your code also handles this case.

Reply:
Please check https://github.com/awslabs/graphstorm/pull/210/commits/bcaaed8ff84692fa3963eea07ec1bf6268663894